### PR TITLE
<Link>Added eloquent javascript

### PR DESCRIPTION
**Pull request by jayk-gupta**
## Changes proposed
Added eloquent javascript in the languages sections

## Screenshots

el

### CODE REVIEW

File: database/languages/javascript.json [hunk 1]
```diff
@@ -12,5 +12,12 @@
         "url": "https://www.javascripttutorial.net/",
         "category": "Other",
         "subcategory": "javascript"
+    },
+    {
+        "name": "Eloquent Javascript",
+        "description": "This is a book about JavaScript,it explains concepts of javascript with real-life examples",
```
Comment (CBID2) [inline database/languages/javascript.json:18]:
```suggestion
        "description": "This  book explains the concepts of JavaScript with real-life examples",
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi there. I fixed the description a bit. Other that, good job! 😊👍🏽

**Comment by jayk-gupta** [comment]
> Hi there. I fixed the description a bit. Other that, good job! 😊👍🏽

Thanks!
I will add more resources in the languages section soon.

File: database/languages/javascript.json (hunk 1 above)
Comment (jayk-gupta) [reply database/languages/javascript.json]: > 

resolved

File: database/languages/javascript.json [hunk 2]
```diff
@@ -16,8 +16,16 @@
...
         "name": "Eloquent Javascript",
         "description": "This  book explains the concepts of JavaScript with real-life examples",
-        "url": "   https://eloquentjavascript.net/",
+        "url": "https://eloquentjavascript.net/",
+        "category": "Other",
+        "subcategory": "javascript"
+    },
+       {
+        "name": "Learn Javascript Online",
+        "description": "Learn javascript in an interactive environment.Read short lessons, take notes, and complete challenges directly in your browser.",
```
Comment (CBID2) [inline database/languages/javascript.json:25]:
```suggestion
        "description": "This website teaches Javascript in an interactive way. You can read short lessons, take notes, and complete challenges directly in your browser.",
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @jayk-gupta! :) I just made a few suggestions in the description section of your PR. You're getting close! 👍🏾 Keep it up! 😄

**Comment by jayk-gupta** [comment]
> Hi @jayk-gupta! :) I just made a few suggestions in the description section of your PR. You're getting close! 👍🏾 Keep it up! 😄

Commited the changes!
Thanks

**Review by CBID2 — APPROVED** [review]
Yay you did it, you did it @jayk-gupta! 😁 LGTM! 👍🏾

**Comment by rupali-codes** [comment]
Thank you @CBID2 and @jayk-gupta :)

**Pull request merged**